### DISCUSSION TITLE
Is there any way to add options after input data

### DISCUSSION THREAD

**Discussion by vmudhire1234** [daniele77/cli#211] [Q&A] [answered]
Hello Daniel, 
Thanks for supporting the CLI.

I would like to check if there is any way to add commands like below

show interface <intf0>
show interface <intf0> stats
show interface <intf0> debug

Here <intf0> is option data. 
Is there any way to add another option after entering option data, like above command. 

Thank you
-venkat

**Comment by daniele77** [comment, marked as answer]
Hi, @vmudhire1234 

You can only get

    show interface           # enter the menu show -> interface
    show interface stats     # command stats inside the menu show -> interface
    show interface debug     # command debug inside the menu show -> interface

where `show` and `interface` are menus and `stats` and `debug` are commands.

The library provides two ways to enter a command:

1. Once inside its menu, enter just the name of the command (e.g., `stats`)
2. From a menu parent of its menu, enter the "full path" of the command (e.g., `show interface stats`)

**Reply by vmudhire1234** [reply]
Thank you
